src/app/cakes/cakes.constants.ts

```diff
@@ -0,0 +1,5 @@
+export const enum Status {
```

Comment:
A better name could be `CakeStatus`.

and a better name for the file will be `cake-status.enum.ts`